<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3840</width>
    <height>2160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendmsg_button">
        <property name="text">
         <string>Send to ALL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="privatemsg_button">
        <property name="text">
         <string>Send private</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="CommonBrowser"/>
      <widget class="QTextBrowser" name="PrivateBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdmin_menu"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_another_Client"/>
    <addaction name="separator"/>
    <addaction name="actionClose_this_Client"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_another_Client">
   <property name="text">
    <string>Open another Client</string>
   </property>
  </action>
  <action name="actionClose_this_Client">
   <property name="text">
    <string>Close this Client</string>
   </property>
  </action>
  <action name="actionAdmin_menu">
   <property name="text">
    <string>Admin Menu</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
